persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxTreeGenerator.java

```diff
@@ -1088,16 +1098,66 @@ public static ImportDeclarationNode getImportDeclarationNode(String orgName, Str
         Token moduleNameToken = AbstractNodeFactory.createIdentifierToken(moduleName);
         SeparatedNodeList<IdentifierToken> moduleNodeList =
                 AbstractNodeFactory.createSeparatedNodeList(moduleNameToken);
+        Token importToken = SYNTAX_TREE_KEYWORD_IMPORT;
+        if (includeAutogeneratedComment) {
+            String generatedComment = COMMENT_GENERATED_BY_BALLERINA;
+            if (!entityName.equals("")) {
```

Comment:
Pass the comment from the calling function. This will reduce complexity. If you have to pass null in some cases, lets have 2 function for these

Reply:
Yes, Now the code is updated to have two functions with comment and without comment and the import prefix is passed to the function as well. [f524212](https://github.com/ballerina-platform/persist-tools/pull/79/commits/f5242122d14dbc8c86354caaecf145c00733a2f7)